packages/react-composites/src/composites/ChatComposite/image-upload/useImageUpload.tsx

```diff
@@ -47,6 +47,24 @@ const getInlineImageData = async (image: string): Promise<Blob | undefined> => {
   return blobImage;
 };
 
+/* @conditional-compile-remove(rich-text-editor-image-upload) */
+/**
+ * @internal
+ * Exported only for unit testing
+ */
+export const removeImageTags = (event: { content: DocumentFragment }): void => {
```

Comment:
@Leah-Xia-Microsoft it looks like the folder should be named ImageUpload instead of image-upload. I think the only place where we use dash for folders name is the folders that have packages names

Reply:
and file-sharing for some reason